Doc/whatsnew/3.11.rst

```diff
@@ -726,6 +726,12 @@ For major changes, see :ref:`new-feat-related-type-hints-311`.
   the given type. At runtime it simply returns the received value.
   (Contributed by Jelle Zijlstra in :gh:`90638`.)
 
+* :data:`typing.TypedDict` subclasses can now be generic. (Contributed by
```

Comment:
`TypedDict` is a function at runtime, and it seems like the [docs](https://docs.python.org/3/library/typing.html#typing.TypedDict) and [PEP 589](https://peps.python.org/pep-0589/#class-based-syntax) studiously avoid the term "subclass" in favour of "class-based syntax" (`TypedDict` never appears in the `__mro__` of a `TypedDict`.)

The docs for `NamedTuple` _do_ use the term "subclass", but I think they probably _shouldn't_, as it leads to confusion like [this](https://github.com/python/mypy/issues/12740).

```suggestion
* :data:`typing.TypedDicts <typing.TypedDict>` can now be generic. (Contributed by
```

Reply:
I used "classes" because I want to make a clear distinction between TypedDict types and instances. But we can just say "types":
```suggestion
* :data:`typing.TypedDict` types can now be generic. (Contributed by
```